src/transport/NetworkProvisioning.h

```diff
@@ -113,6 +113,14 @@ class DLL_EXPORT NetworkProvisioning
      */
     CHIP_ERROR SendIPAddress(const Inet::IPAddress & addr);
 
+    /**
+     * @brief
+     *  The device can use this function to send its current IP address to
+     *  commissioner. This would generally be called during network
+     *  provisioning of the device, when the device already has an IP address.
+     */
```

Comment:
Should we add a GH issue to change this?
If this is to stay, the comment should explain why we are specifically using IPv4 when CHIP is supposed to work with IPv6.

Reply:
@andy31415, this code `NetworkProvisioning.h/.cpp` will be deleted once we move the provisioning to ZCL cluster. I'll file a GH issue in regards to that.

Reply:
Filed #4606